### ISSUE TITLE
Noise Level in AddImpulseResponse

### ISSUE THREAD

**Issue by shakeel608** [iver56/audiomentations#97]
How do you define the noise level in class `AddImpulseResponse` 
Say after getting impulse response from a dateset (Impulse responses can be created using e.g. http://tulrich.com/recording/ir_capture/), 

How do you define  noise level for adding Impulse response in this ?

**Comment by iver56** [comment]
I don't understand the question. What does noise level have to do with room impulse response convolution? Can you rephrase?

**Comment by iver56** [comment]
Is it the name that is confusing? The transform isn't actually "adding" an impulse response waveform, it's convolving the input with it, typically to give it more reverb

**Comment by iver56** [comment]
Related: https://github.com/iver56/audiomentations/issues/56
